src/btree/bt_sync_obsolete.c

```diff
@@ -406,7 +406,8 @@ __checkpoint_cleanup_walk_btree(WT_SESSION_IMPL *session, WT_ITEM *uri)
     /* Open a handle for processing. */
     ret = __wt_session_get_dhandle(session, uri->data, NULL, NULL, 0);
     if (ret != 0)
```

Comment:
We should handle `ret` returning an error not being `EBUSY now and error out if that's the case.

Reply:
Thanks. Missed to return the `ret` to the caller. The caller already handles the error type.